### PR TITLE
DOC: finetune_depth has its won separate tutorial

### PR DESCRIPTION
**Pull request by marcopeix**
Make a standalone tutorial for `finetune_depth`.
Remove the `finetune_depth` portion of the fine-tuning tutorial.

### CODE REVIEW

File: nbs/docs/tutorials/23_finetune_depth_finetuning.ipynb [hunk 1]
```diff
@@ -0,0 +1,432 @@
+{
```
Comment (AzulGarza) [inline nbs/docs/tutorials/23_finetune_depth_finetuning.ipynb:1]: <p>thank you Marco! could you add a small summary on the meaning of <code>finetune_depth</code>?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (marcopeix) [reply nbs/docs/tutorials/23_finetune_depth_finetuning.ipynb:1]: Done, let me know what you think!

**Review by tracykteal — APPROVED** [review]
Thanks @marcopeix. LGTM!

**Comment by jmoralez** [comment]
Please remember to add labels to the PRs. This also needs fixing the typo in the title when creating the release notes.

**Comment by tracykteal** [comment]
Thanks, I just fixed the title.

**Comment by jmoralez** [comment]
It's too late now, the PR title gets used as the commit message and it shows up like that in the [draft release](https://github.com/Nixtla/nixtla/releases). We'll have to update it there when publishing the release.
